### ISSUE TITLE
Create Landing Page for Pizza Flavors Site

### ISSUE THREAD

**Issue by JABt3ch** [JABt3ch/Pizza-Flavors#1]
I am working with open source website for different Pizza Flavors. This is part of Hacktoberfest 2019.

**Comment by NuCelyA** [comment]
@JABt3ch  i would like to work on that

**Comment by stnbnvdz** [comment]
Will contribute on this.

**Comment by JABt3ch** [comment]
@NuCelyA yes. you can work on About page if you like. Please put your own styles

**Comment by JABt3ch** [comment]
> Will contribute on this.

Work on menu as I will work on landing page.
